recipes/newrelic/mongodb/debian.yml

```diff
@@ -0,0 +1,247 @@
+  - name: "NR_CLI_DB_HOSTNAME"
+    prompt: "Please enter your MongoDb hostname below. If none is provided the default value: localhost will be used."
+    default: "localhost"
+  - name: "NR_CLI_DB_PORT"
+    prompt: "Please enter your MonogDb PORT below. If none is provided the default value: 27017 will be used."
+    default: 27017
+  - name: "NR_CLI_SSL"
+    prompt: "Are you using SSL? (y/n)"
+  - name: "NR_CLI_CERT_AUTH_FILE"
+    prompt: "Please enter the path to your Certificate Authority file."
+  - name: "NR_CLI_SKIP_SSL_VERIFY"
+    prompt: "Would you like to verify the certificate sent by the host? (y/n)"
+  - name: "NR_CLI_CLIENT_CERT_FILE"
+    prompt: "Please enter the path to your Client Certificate file, if required."
+  - name: "NR_CLI_CERT_PASSPHRASE"
+    prompt: "Please enter your passphrase to decrypt the PEM key file, if required."
+
+install:
+
+  version: "3"
+  silent: true
+
+  tasks:
+    default:
+      cmds:
+        - task: create_user_notification
+        - task: setup
+        - task: restart
+        
+    create_user_notification:
+      cmds:
+        - |
+          echo "_ Note: These steps must be completed in order for the MongoDB integration to function properly.
+          In the MongoDB shell, enter use admin.
+          Use the following command to create the listCollections role.
+          db.createRole({role: "listCollections", privileges: [{ resource: {db:"",collection:""}, actions: ["listCollections"] }], roles: []})
+          Use the following commands to create a new user, and assign clusterMonitor and listCollections roles to the user:
+          db.createUser({ user: "username", pwd: "password", roles: [ "clusterMonitor", "listCollections" ]})"
+
+
+    assert_pre_req:
+      cmds:
+        - |
+          SERVICE_EXIST=$(sudo systemctl status newrelic-infra.service | grep "Active" | wc -l)
+          if [ $SERVICE_EXIST -eq 0 ]; then
+            echo "The newrelic-infra agent service is NOT installed on the host, but is required to install this integration." >> /dev/stderr
+            exit 1
+          fi
+        - |
+          MONGODB_EXISTS=$(mongo --eval 'db.runCommand({ connectionStatus: 1 })' | grep "ok" | awk '{print $1 $2 $3}')
+          if [ $MONGODB_EXISTS != "ok:1"]; then
+            echo "Cannot estabilish connection to mongodb, please verify that it's installed correctly."
+            exit 2
+          fi
+
+    setup:
+      label: "Installing mongodb integration..."
+      cmds:
+        - |
+          sudo mkdir -p "/etc/newrelic-infra/integrations.d"
+        - |
+          sudo apt-get update
+        - |
+          sudo apt-get install nri-mongodb -y
+        - |
+          if [ -f /etc/newrelic-infra/integrations.d/mongodb-config.yml ]; then
+            sudo rm /etc/newrelic-infra/integrations.d/mongodb-config.yml;
+          fi
+
+          sudo cp /etc/newrelic-infra/integrations.d/mongodb-config.yml.sample /etc/newrelic-infra/integrations.d/mongodb-config.yml;
```

Comment:
You can remove this cp command, since the file gets written with new content below.